website/templates/search.mako

```diff
@@ -235,8 +235,12 @@
       <div data-bind="template: {name: 'node', data: $data}"></div>
     </script>
     <script type="text/html" id="registration">
-        <h4><a data-bind="attr.href: url">{{ title }}</a>  (<!-- ko if: is_retracted --><span class="text-danger">Retracted</span> <!-- /ko -->Registration)</h4>
-
+        <!-- ko if: parent_url -->
+        <h4><a data-bind="attr.href: parent_url">{{ parent_title}}</a> / <a data-bind="attr.href: url">{{ title }}</a>  (<!-- ko if: is_retracted --><span class="text-danger">Retracted</span> <!-- /ko -->Registration)</h4>
+        <!-- /ko -->
+        <!-- ko if: !parent_url -->
+        <h4><span data-bind="if: parent_title">{{ parent_title }} /</span> <a data-bind="attr.href: url">{{ title }}</a>  (<!-- ko if: is_retracted --><span class="text-danger">Retracted</span> <!-- /ko -->Registration)</h4>
```

Comment:
This line of code uses two different form of knockout syntax for the `if` statement; I'm following up with @sloria to see if there are any style guidelines here, but this seems unnecessarily inconsistent.